### ISSUE TITLE
CalDAV calendar & tasks write issue with Thunderbird 78

### ISSUE THREAD

**Issue by sbaumgartner75** [tine20/tine20#7198]
According to https://www.thunderbird.net/en-US/thunderbird/78.0/releasenotes/ "Lightning" has been removed from the User-Agent. Thus Thunderbird is no longer detected as supported Client as it can be seen in the log:

ac2f65 - baumi - 2020-08-02T05:35:56+02:00 DEBUG (7): Calendar_Convert_Event_VCalendar_Factory::parseUserAgent::172 Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:78.0) Gecko/20100101 Thunderbird/78.1.0 ->  backend: generic version:

I temporarily fixed it by adding "Thunderbird" to HEADER_MATCH in the following files:

````
root@office:/var/www/html# grep -R Lightning * | grep HEADER_MATCH
Calendar/Convert/Event/VCalendar/Thunderbird.php:    const HEADER_MATCH = '/ (Thunderbird|Lightning|Iceowl)\/(?P<version>\S+)/';
Tasks/Convert/Task/VCalendar/Thunderbird.php:    const HEADER_MATCH = '/ (Thunderbird|Lightning|Iceowl)\/(?P<version>\S+)/';

```

I will open a pull request.

**Comment by pschuele** [comment]
hi, thanks for the PR - I'll review it now.

**Comment by pschuele** [comment]
fixed by the PR #7199
